src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/controller/WelcomeControllerIntegrationTest.java

```diff
@@ -125,93 +146,439 @@ public void welcomeApiTest() throws Exception {
 
 
     @Test
-     public void createOrgRoleMappingTest() throws Exception {
+    public void createOrgRoleMappingTest() throws Exception {
+        Mockito.when(crdFeignClientFallback.createRoleAssignment(any()))
+                .thenReturn(new ResponseEntity<>(IntTestDataBuilder
+                        .buildListOfUserProfiles(true, false,"1", "2", ROLE_NAME_STCW, ROLE_NAME_TCW,
+                                true, true, false,
+                                true,"BFA1", "BFA2",false), HttpStatus.OK));
+
+        UserRequest request = UserRequest.builder()
+                .users(Arrays.asList("21334a2b-79ce-44eb-9168-2d49a744be9z", "21334a2b-79ce-44eb-9168-2d49a744be9x"))
+                .build();
+        logger.info(" createOrgRoleMappingTest...");
+        String uri = "/am/role-mapping/staff/users";
+        setRoleAssignmentWireMock(HttpStatus.CREATED, SAMPLE_RAS_RESPONSE);
+
+        mockMvc.perform(post(uri)
+                .contentType(JSON_CONTENT_TYPE)
+                .headers(getHttpHeaders())
+                .content(mapper.writeValueAsBytes(request)))
+                .andExpect(status().is(200))
+                .andReturn();
+    }
+
+    @Test
+    @DisplayName("S1: must successfully create org role mapping for single user with one role assignment")
+    public void createOrgRoleMappingForSingleUserWithOneRoleAssignment() throws Exception {
+
+        Mockito.when(crdFeignClientFallback.createRoleAssignment(any()))
+                .thenReturn(new ResponseEntity<>(IntTestDataBuilder
+                        .buildListOfUserProfiles(false, false,"1", "2", ROLE_NAME_STCW, ROLE_NAME_TCW,
+                                true, true, false,
+                                true,"BFA1", "BFA2",
+                                false), HttpStatus.OK));
+
+        UserRequest request = UserRequest.builder()
+                .users(Arrays.asList("123e4567-e89b-42d3-a456-556642445674"))
+                .build();
+        logger.info(" createOrgRoleMappingTest...");
+        String uri = "/am/role-mapping/staff/users";
+        setRoleAssignmentWireMock(HttpStatus.CREATED, RAS_ONE_USER_ONE_ROLE);
+
+        MvcResult result = mockMvc.perform(post(uri)
+                .contentType(JSON_CONTENT_TYPE)
+                .headers(getHttpHeaders())
+                .content(mapper.writeValueAsBytes(request)))
+                .andExpect(status().is(200))
+                .andReturn();
+
+        String contentAsString = result.getResponse().getContentAsString();
+        assertTrue(contentAsString.contains("123e4567-e89b-42d3-a456-556642445674"));
+        assertTrue(contentAsString.contains("APPROVED"));
```

Comment:
we need to follow above assertion and parse the response to check individual assignments.